src/core/evaluator.js

```diff
@@ -318,14 +315,10 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
           groupOptions.isolated = (group.get('I') || false);
           groupOptions.knockout = (group.get('K') || false);
           if (group.has('CS')) {
-            colorSpace = group.get('CS');
-            if (colorSpace) {
-              colorSpace = ColorSpace.parse(colorSpace, this.xref, resources,
-                                            this.pdfFunctionFactory);
-            } else {
-              warn('buildFormXObject - invalid/non-existent Group /CS entry: ' +
-                   group.getRaw('CS'));
-            }
+            colorSpace = await this.parseColorSpace({
+              cs: group.get('CS'),
+              resources,
+            });
```

Comment:
This is essentially a better, and more general, version of PR 11047; as suggested in https://github.com/mozilla/pdf.js/pull/11047#discussion_r310626220